package/MDAnalysis/transformations/__init__.py

```diff
@@ -53,7 +53,30 @@ def wrapped(ts):
 
 See `MDAnalysis.transformations.translate` for a simple example.
 
+To meet the need of serialization of universe, transformations are converted
```

Comment:
The conversion is not really relevant to a user of the library – they deal with what's there. You can add a comment to the code talking about the thinking and motivation or you can write a specific developer section. But as I said below, the text needs to make sense to someone who comes to the code without the historical knowledge. Try to put yourself into the shoes of someone who sees this the first time.